src/service/avatar.ts

```diff
@@ -63,9 +63,7 @@ export class AvatarMetadata {
       if (typeof error === 'string') {
         console.log(`${this.uri} - error:`, error);
       }
-      if (error === 'Image is not available') {
-        throw new RetrieveURIFailed(error, 404);
-      }
+      throw new RetrieveURIFailed(`Error fetching avatar: Provided url or NFT source is broken.`, 404);
```

Comment:
It would be helpful if the error message provide the URL it fetch the image from

Reply:
the call happens under ens-avatar, and not all the errors contain the URL to be shown. I will make the improvement on that side first and update the metadata later.